tensorflow/core/distributed_runtime/rpc/grpc_server_lib.h

```diff
@@ -127,6 +127,8 @@ class GrpcServer : public ServerInterface {
   void set_channel_cache(GrpcChannelCache* channel_cache) {
     channel_cache_.reset(channel_cache);
   }
+ protected:
+  WorkerEnv worker_env_;
```

Comment:
Can you call worker_env() instead of changing worker_env_ visibility?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/distributed_runtime/rpc/grpc_server_lib.h#L95

Reply:
fixed, thx